mathesar/models/query.py

```diff
@@ -91,33 +112,53 @@ def _validator(jp_path):
                         )
                         raise InvalidValueType(
                             message,
-                            field=field_name,
+                            field=self.field_name,
                         )
-    return _validator
 
+    def __eq__(self, other):
+        return (
+            isinstance(other, JpPathValidator) and self.field_name == other.field_name and self.jp_path == other.jp_path
```

Comment:
Is `and self.jp_path == other.jp_path` a mistake? Doesn't seem to be defined, and I'm not sure what the intent could be.

Reply:
mybad its a mistake